docs/ADL2/master01-preface.adoc

```diff
@@ -65,6 +65,26 @@ For existing users of ADL or archetype development tools, the following provides
 
 The object syntax used to represent the description, terminology and annotations sections of an ADL archetype has been historically known as 'dADL' (i.e. 'data ADL'). Since this syntax is completely generic and has no specific dependency on either ADL or openEHR, it has been separated out into its own specification known as Object Data Instance Notation (ODIN).
 
+=== ADL 2.4
+
+==== Changes
```

Comment:
SEC also decided to explore deprecating odin in favour of yaml/json. Let's decide on leaving that out of this version. And keep it for a next release. This comment is just to formalise that decision and not to forget.
But a comment on the ODIN section would be helpful.

Reply:
The latest discussions on discourse leave me wondering whether we're really in favour of abandoning ODIN, or just accept it as something that works and does not make sense to change at this point.